<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GuitarText</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/file/icon/icon.png</normaloff>:/file/icon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>254</red>
          <green>254</green>
          <blue>254</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>254</red>
          <green>254</green>
          <blue>254</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>106</red>
          <green>104</green>
          <blue>100</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>190</red>
          <green>190</green>
          <blue>190</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>239</red>
          <green>235</green>
          <blue>231</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <widget class="QMenu" name="menuLast_open">
     <property name="title">
      <string>Ostatnio otwarte</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuLast_open"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Narzędzia</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNext_Chords"/>
    <addaction name="actionPrevious_Chords"/>
    <addaction name="separator"/>
    <addaction name="actionNext_schift_chords"/>
    <addaction name="actionPrevious_schift_chords"/>
    <addaction name="separator"/>
    <addaction name="actionFull_Screen"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionProperties"/>
    <addaction name="actionQuickOptions"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionGuitarText"/>
   </widget>
   <widget class="QMenu" name="menuFavourite">
    <property name="title">
     <string>Ulubione</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuFavourite"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionSearch"/>
   <addaction name="actionQuickOptions"/>
   <addaction name="actionProperties"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/new.png</normaloff>:/file/icon/new.png</iconset>
   </property>
   <property name="text">
    <string>Nowy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/open.png</normaloff>:/file/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Otwórz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/save.png</normaloff>:/file/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/saveas.png</normaloff>:/file/icon/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz jako</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/refresh.png</normaloff>:/file/icon/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Wczytaj ponownie</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/exit.png</normaloff>:/file/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Wyjście</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNext_Chords">
   <property name="text">
    <string>Następna wersja akordów</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="actionPrevious_Chords">
   <property name="text">
    <string>Poprzednia wersja akordów</string>
   </property>
   <property name="toolTip">
    <string>Previous Chords</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>O Qt</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pełny ekran</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionNext_schift_chords">
   <property name="text">
    <string>Transpozycja w górę</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Right</string>
   </property>
  </action>
  <action name="actionPrevious_schift_chords">
   <property name="text">
    <string>Transpozycja w dół</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Left</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/propertise.png</normaloff>:/file/icon/propertise.png</iconset>
   </property>
   <property name="text">
    <string>Właściwości</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/search.png</normaloff>:/file/icon/search.png</iconset>
   </property>
   <property name="text">
    <string>Wyszukaj w bazie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionGuitarText">
   <property name="text">
    <string>O GuitarText</string>
   </property>
  </action>
  <action name="actionQuickOptions">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/file/icon/quick.png</normaloff>:/file/icon/quick.png</iconset>
   </property>
   <property name="text">
    <string>Szybkie opcje</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
